Week3/homework/step2-3.js

```diff
@@ -2,46 +2,47 @@
 
 // Use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
-  // eslint-disable-next-line prefer-const
-  let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
-  return result;
+	// eslint-disable-next-line prefer-const
+	const result = " ";
```

Comment:
Ah, this seems to be a bug. Don't think this will work. `const` does not allow the variable to be reassigned which is what is happening below.